widgets/centreon-widget-service-monitoring/service-monitoring/src/index.php

```diff
@@ -370,7 +371,7 @@
         AND acl.service_id = s.service_id
         AND acl.group_id IN (" . $groupList . ") ";
 }
-$orderBy = 'hostname ASC , description ASC';
+$orderBy = 'hostname ASC ';
```

Comment:
we accept the fact that services are not ordered anymore ?
Example, it will be harder to find a service if the host has 20 services

Reply:
yes,
only default order changes. 
An user could steel change default order and override with it's own order